master/internal/templates/api_intg_test.go

```diff
@@ -150,7 +150,7 @@ func TestGetTemplates(t *testing.T) {
 
 	t.Run("GetTemplates filter by workspace", func(t *testing.T) {
 		resp, err := api.GetTemplates(ctx, &apiv1.GetTemplatesRequest{
-			WorkspaceId: workspaceIDs[0],
+			WorkspaceIds: []int32{workspaceIDs[0]},
```

Comment:
Is it worth unit testing with multiple workspace IDs? Is it possible one workspace ID could be invalid? (I wouldn't think so ...)